public/ConvertFrom-TNRestResponse.ps1

```diff
@@ -156,7 +156,7 @@
                 if ('Id' -in $keys) {
                     $null = $order.Add("Id")
                 }
-                if ($Type) {
+                if ('Type') {
```

Comment:
This will always return $true. You can test it from your command line

```
if ('Type') { write-warning hello }
WARNING: hello
```

Instead, change it back to `$Type` and perhaps pass a `-Type` elsewhere, likely in `Save-TNReportResult`.